src/tasks/noncombat.ts

```diff
@@ -26,7 +26,10 @@ import { handleCustomPulls, logTestSetup, tryAcquiringEffect, wishFor } from "..
 import { CombatStrategy } from "grimoire-kolmafia";
 import Macro from "../combat";
 
-const comTestMaximizerString = "-combat";
+const haveTurkey = have($familiar`Peace Turkey`);
+const rate = haveTurkey ? 0.2 : 0.133; // If we have peace turkey, familiar weight scales better
+const cap = haveTurkey ? 50 : 75; // If we have peace turkey, more than 50 lbs isn't helpful; 75 for disgeist
+const comTestMaximizerString = `-combat, ${rate} familiar weight ${cap} max`;
```

Comment:
Where do the numbers for `rate` come from? 50lbs is worth 10NC on the turkey, so should the rate not be `5`lbs : 1NC instead of `1/5`lbs : 1 NC?

Reply:
My understanding of the maximizer function was that weighting this way meant 5 familiar weight was worth 1 point (1 NC) but I don't actually know how maximizer treats NC holistically. Regardless, with a more recent mafia version this is unnecessary.